build/lib/extensions.ts

```diff
@@ -44,7 +44,12 @@ export function packageBuiltInExtensions() {
 		.filter(({ name }) => builtInExtensions.every(b => b.name !== name))
 		.filter(({ name }) => sqlBuiltInExtensions.indexOf(name) >= 0);
 	const visxDirectory = path.join(path.dirname(root), 'vsix');
-	fs.mkdirSync(visxDirectory);
+	try {
```

Comment:
would it make more sense to just rimraf the directory or check for exists before trying to make?

Reply:
I think it's safe to just ignore errors here.  If something important went wrong the next steps will fail and I don't have to keep updating this script for future possible issues (e.g. a file is locked, etc.)

Reply:
For context, this is only encountered when generating the packaged builds in a dev environment multiple times.

Reply:
If its locked won't later steps fail though?

Reply:
yeah, maybe, and we can get an error therr

Reply:
..I'll add a check `if exists` to prevent logging the warning message unnecessarily, but I'll keep the try\catch to avoid having to update the script if it fails for some other reason.